packages/bot-web-ui/src/components/quick-strategy/selects/symbol.tsx

```diff
@@ -33,13 +34,19 @@ type TSymbolSelect = {
 
 const SymbolSelect: React.FC<TSymbolSelect> = ({ fullWidth = false }) => {
     const { quick_strategy } = useDBotStore();
+    const {
+        ui: { is_mobile, is_desktop },
+    } = useStore();
     const { setValue } = quick_strategy;
     const [active_symbols, setActiveSymbols] = React.useState([]);
+    const [is_input_started, setIsInputStarted] = useState(false);
+    const [input_value, setInputValue] = useState('');
     const { setFieldValue, values } = useFormikContext<TFormData>();
 
     React.useEffect(() => {
```

Comment:
@maryia-matskevich-deriv can we import useEffect as well as we did it for useState?
lets keep one approach throughout the components.

``` suggestion
useEffect(() => {
```

Reply:
✅done